package.json

```diff
@@ -319,8 +319,8 @@
 		"storybook:e2e:dev": "concurrently \"npm run dev:packages\" \"storybook dev -c test/storybook-playwright/storybook -p 50241\"",
 		"test": "npm-run-all lint test:unit",
 		"test:create-block": "bash ./bin/test-create-block.sh",
-		"test:e2e": "wp-scripts test-e2e --config packages/e2e-tests/jest.config.js",
-		"test:e2e:debug": "wp-scripts --inspect-brk test-e2e  --runInBand --no-cache --verbose --config packages/e2e-tests/jest.config.js --puppeteer-devtools",
+		"test:e2e": "wp-scripts test-playwright --config test/e2e/playwright.config.ts",
+		"test:e2e:debug": "wp-scripts test-playwright --config test/e2e/playwright.config.ts --ui",
 		"test:e2e:playwright": "wp-scripts test-playwright --config test/e2e/playwright.config.ts",
```

Comment:
Should these `e2e:playwright` scripts also be deleted? If they're being left for backwards compatibility, can they just call `test:e2e` and `e2e:debug` to avoid having the commands written out twice?